docs/operating-system/debugging.md

```diff
@@ -17,13 +17,18 @@ SSH access through the [SSH add-on] (which will give you SSH access through port
 
 ### Home Assistant Operating System
 
-Use a USB drive formatted with FAT, ext4, or NTFS and name it CONFIG (case sensitive). Create an `authorized_keys` file (no extension) containing your public key, and place it in the root of the USB drive. File needs to be ANSI encoded (not UTF-8) and must have Unix line ends (LF), not Windows (CR LF). See [Generating SSH Keys](#generating-ssh-keys) section below if you need help generating keys. Use the CLI (eg. SSH to the [SSH add-on] on port 22) and import the `authorized_keys` file with the `ha os import` command. You can now access your device as root over SSH on port 22222. Alternatively, the file will be imported from the USB when the Home Assistant OS device is rebooted.
+Using a USB drive formatted with FAT32 or ext4 and named CONFIG (case sensitive), follow the following steps:
+
+1. Create an ANSI-encoded (not UTF-8) file named `authorized_keys` (no extension) containing your public key. Ensure that the file has Unix line endings (LF) and not Windows line endings (CR LF). See section [Generating SSH Keys](#generating-ssh-keys) below for instructions on generating keys.
+1. Place the `authorized_keys` file in the root of the USB drive.
+1. Use the CLI (e.g., SSH to the [SSH add-on] on port 22) to import the `authorized_keys` file by issuing command `ha os import`. Alternatively, the file will be imported from the USB when the Home Assistant OS device is rebooted.
+1. SSH root access to your host should now be available on port 22222.
```

Comment:
Presumably the above should be incrementing numbers?

Reply:
In theory, it doesn't actually matter. Some people suggest doing it like this, knowing it will be fixed on render, and therefore it's easier to add items, without renumbering (or leaving it out of order which actually doesn't matter on render, but will look weird raw)

https://www.markdownguide.org/basic-syntax/#ordered-lists